feat: migration guide for Note and Notification (#1497)
contentful/forma-36: MIGRATION.md

  <GridItem />
</Grid>;
```

## Note

> API of Note component was changed slightly, mostly around properties that are now aligned with our code [styleguide](https://github.com/contentful/forma-36/blob/forma-v4/docs/code-style-guide.md).

Reviewer: ```suggestion
The properties of the Note component API have been aligned with our code [style guide](https://github.com/contentful/forma-36/blob/forma-v4/docs/code-style-guide.md).
```

### How to migrate your Note components

> Run the [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod) to migrate your v3 `Note` component to the new version:

Reviewer: ```suggestion
To migrate your `Note` component to v4, run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):
```

`npx @contentful/f36-codemod`

> If you decide to do it manually, you would need to transform you existing code:

Reviewer: ```suggestion
If you want to do it manually, you must transform your existing code as follows:
```

```tsx
import { Note } from '@contentful/forma-36-react-components';

<Note noteType="positive">
  A piece of information that is relevant to the context the user is currently
  in.
</Note>;

<Note noteType="negative" hasCloseButton>
  A piece of information that is relevant to the context the user is currently
  in.
</Note>;

<Note
  title={'Title example'}
  noteType="primary"
  hasCloseButton
  onClose={() => {}}
>
  A piece of information that is relevant to the context the user is currently
  in.
</Note>;
```

into:

```tsx
import { Note } from '@contentful/f36-components';

<Note variant="positive">
  A piece of information that is relevant to the context the user is currently
  in.
</Note>;

<Note variant="negative" withCloseButton>
  A piece of information that is relevant to the context the user is currently
  in.
</Note>;

<Note
  title={'Title example'}
  variant="primary"
  withCloseButton
  onClose={() => {}}
>
  A piece of information that is relevant to the context the user is currently
  in.
</Note>;
```

## Notification

> API of Notification component was changed slightly, mostly around properties that are now aligned with our code [styleguide](https://github.com/contentful/forma-36/blob/forma-v4/docs/code-style-guide.md).

Reviewer: ```suggestion
The properties of the Notification component API have been aligned with our code [style guide](https://github.com/contentful/forma-36/blob/forma-v4/docs/code-style-guide.md).
```

> ### How to migrate your Note components

Reviewer: ```suggestion
### How to migrate your Notification components
```

> Run the [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod) to migrate your v3 `Note` component to the new version:

Reviewer: ```suggestion
To migrate your 'Notification' component to v4, run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):
```

`npx @contentful/f36-codemod`

> If you decide to do it manually, you would need to transform you existing code:

Reviewer: ```suggestion
If you want to do it manually, you must transform your existing code as follows:
```
